content/communities/uswds.md

```diff
@@ -36,10 +40,6 @@ community_list:
     members: 108
     emails_per_week:
 
-redirectto: https://designsystem.digital.gov/about/community/
```

Comment:
keep this `redirectto:` It is was redirects the page to the design system website since we no longer support it as a community.